sbin/jtreg.sh

```diff
@@ -22,12 +26,20 @@ JTREG_VERSION=${JTREG_VERSION:-4.2.0-tip}
 JTREG_TARGET_FOLDER=${JTREG_TARGET_FOLDER:-jtreg}
 JOB_NAME=${JOB_NAME:-OpenJDK}
 NUM_PROCESSORS=${NUM_PROCESSORS:-$(getconf _NPROCESSORS_ONLN)}
+TMP_DIR=/tmp/
```

Comment:
use `$TMPDIR` (non-windows) and `$TEMP` (Windows)
Will `$TMPDIR` work on other than Linux?